cves/CVE-2013-0856.yml

```diff
@@ -146,10 +154,10 @@ unit_tested:
     In FFmpeg, they have the FATE suite of regression tests. They are not
     "unit" tests per se, but are automated regression tests. If the fix for the
     vulnerability involved adding or updating a FATE suite, then make fix: true.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: 'code in and around is not unit tested.'
```

Comment:
code coverage from the lines of code from the 'fix' commit suggest that unit tests may be involved. However the code coverage only shows coverage from a recent build, it does not show what happened years ago. http://coverage.ffmpeg.org/index.src_libavcodec_alac.c.html

Might be worth mentioning

Reply:
Ahh i missed that, thank you. Ill add that when I resubmit for submission 2